fab

```diff
@@ -17,17 +17,18 @@ from typing import (
 )
 
 try:
-    from typing import Protocol # type: ignore
-except:
-    from typing_extensions import Protocol # type: ignore
+    from typing import Protocol  # type: ignore
+except ModuleNotFoundError:
+    from typing_extensions import Protocol  # type: ignore
```

Comment:
Low Priority:

I personally use `ImportError` for situations like these rather than `ModuleNotFoundError`, so that regardless of the *kind* of error importing it will still try the second import. This isn't an issue atm, but *might* be in the future (afaik `ModuleNotFoundError` is currently the only subclass of `ImportError`, see https://docs.python.org/3/library/exceptions.html#exception-hierarchy).

Reply:
Awesome, thanks dude. I followed your advice.